pages/common/while.md

```diff
@@ -0,0 +1,11 @@
+# while
+
+> Simple shell loop
+
+- Execute a command forever once every second.
```

Comment:
It would be gentler to start with a terminating while example before this one. "Forever" is scary for beginners :)